pkg/internal/controller/controller.go

```diff
@@ -228,11 +229,16 @@ func (c *Controller) reconcileHandler(obj interface{}) bool {
 		return true
 	}
 
+	ctx := reconcile.Context{
+		Context: context.Background(),
+		Log:     c.Log.WithValues("name", req.Name, "namespace", req.Namespace),
```

Comment:
also add gvk information? and/or reconciler name?

Reply:
The reconciler/controller name is already within the c.Log logger, +1 adding gvk info